Tasks/RunLoadTestV1/Start-CloudLoadTest.ps1

```diff
@@ -233,5 +233,13 @@ else
 	Write-Error ("Connection '{0}' failed for service '{1}'" -f $connectedServiceName, $connectedServiceDetails.Url.AbsoluteUri)
 }
 
+$featureFlags = @{
+	failDeprecatedBuildTask  = [System.Convert]::ToBoolean($env:FAIL_DEPRECATED_BUILD_TASK)
+}
+if ($featureFlags.failDeprecatedBuildTask)
+{
+	throw "The CloudLoadTest@1 (Cloud-based load test) has been deprecated since June 4, 2019 and will soon be retired. Use the AzureLoadTest@1 task instead."
```

Comment:
Shouldn't this string be sourced from task.json?

Reply:
Same as above